style.css

```diff
@@ -44,142 +20,192 @@ h1 {
-}
-#select-input-container {
-  display: flex;
-  justify-content: center;
-}
-/*results */
-#results {
-  color: rgb(27, 26, 26);
-  font-size: 1.2rem;
-}
-.genre-rating-status {
-  height: 100%;
-  background-color: rgb(235, 229, 220);
-  font-size: 18px;
-  padding-bottom: 0px;
-  margin-top: 10px;
-  text-align: left;
-  border-radius: 5px;
-  display: block;
-  text-align: justify;
+#root > div > div {
+  width: 100%;
+  height: 30%;
+  border-radius: 10px;
+  margin-top: 30px;
+  font-size: larger;
+  color: orange;
 }
-
 @media only screen and (max-width: 400px) {
-  html,
-  body {
-    overflow-x: hidden;
-  }
   #root {
     color: black;
-    background-color: rgb(245, 239, 239);
+    background-color: #164a41;
     display: grid;
     grid-template-columns: repeat(1, 380px);
-    gap: 0px;
+    gap: 10px;
     grid-auto-rows: max-content;
     width: 100%;
   }
   h1 {
     font-family: sans-serif;
     font-size: 20px;
     text-align: left;
-    color: rgb(15, 15, 15);
+    color: white;
   }
-  .title-image-summary-container {
+  #root > div {
+    width: 100%;
+    height: 75%;
+    background-color: #164a41;
     margin-top: 20px;
-    background-color: rgb(219, 197, 179);
+  }
+  .episode-title {
+    width: 90%;
+    text-align: center;
+    margin-top: 0px;
+    height: 30px;
+    background-color: rgb(247, 154, 93);
+    border: 1px solid rgb(160, 159, 159);
     border-radius: 10px;
-    overflow: hidden;
-    box-shadow: 2px 5px grey;
-    width: 100%;
-    padding-left: 0px;
-    margin-left: 0px;
-    margin-right: 0px;
+    padding-top: 15px;
+    box-shadow: 2px 2px grey;
   }
   .episode-image {
-    height: 250px;
-    width: 280px;
+    height: 30%;
+    width: 50%;
+    margin-left: 2%;
+    margin-right: 2%;
+  }
+  .episode-summary {
+    width: 90%;
+    color: white;
   }
   #select-input-container {
     display: flex;
     justify-content: flex-start;
   }
   #search-field {
-    height: 35px;
+    height: 20px;
     width: 120px;
-    background-color: rgb(252, 223, 162);
-    border-radius: 5px;
     font-size: 1rem;
-    outline-color: grey;
-    box-shadow: 2px 2px rgb(146, 144, 146);
   }
 
   #selectMenu {
-    background-color: rgb(252, 223, 162);
-    border-radius: 5px;
+    background-color: white;
     font-size: 1rem;
-    outline-color: grey;
-    border: 2px solid grey;
-    box-shadow: 2px 2px rgb(146, 144, 146);
     width: 120px;
-    height: 35px;
+    height: 20px;
     margin-right: 10px;
   }
   /*select show*/
   #selectShow {
-    background-color: rgb(252, 223, 162);
-    border-radius: 5px;
+    background-color: white;
     font-size: 1rem;
-    outline-color: grey;
-    border: 2px solid grey;
-    box-shadow: 2px 2px rgb(146, 144, 146);
     width: 120px;
-    height: 35px;
+    height: 20px;
     margin-right: 10px;
   }
+  #root > div > div {
+    width: 90%;
+    height: 30%;
+  }
 }
+/* media query for over 400px */
 @media only screen and (min-width: 400px) {
```

Comment:
I like that you added media queries. It improves the aesthetic qualities of your site.